### PR TITLE
clarify

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/issues/317

@tseward @dariomws what do you think?

### CODE REVIEW

**Review by tseward — COMMENTED** [review]
You indicate 'installed on the same computer' and then go to reference SharePoint Server. Is it the same computer (yes) or exclusive to SharePoint Server (no).

**Comment by get-itips** [comment]
Well, the same computer text was already there, what I want to express is the following:

regular computer with client components sdk -> safe to uninstall

SharePoint server  -> do not uninstall as you will break SP

What do you think?

**Review by dariomws — COMMENTED** [review]
If SharePoint Client Components SDK is included in every installation of SharePoint Server because it's a required component, I think we can add a new note saying that SPO Management Shell is not supported in SP servers.
Anyway, I think this is a very special situation to install the SPO MS in an SP server...

File: sharepoint/docs-conceptual/sharepoint-online/connect-sharepoint-online.md [hunk 1]
```diff
@@ -70,4 +71,4 @@ To assign a user the SharePoint admin role, see [Assign admin roles](/microsoft-
 You are now ready to use SharePoint Online commands.
 
 > [!NOTE]
-> There is a known issue between the SharePoint Online Management Shell module and SharePoint Client Components SDK where the module will fail to load if both are installed on the same computer. If you encounter this issue, uninstall the SharePoint Client Components SDK.
+> There is a known issue between the SharePoint Online Management Shell module and SharePoint Client Components SDK where the module will fail to load if both are installed on the same computer. If you encounter this issue, and you are not trying to use the SharePoint Online Management Shell module in a SharePoint Server, uninstall the SharePoint Client Components SDK.
```
Comment (JohanFreelancer9) [inline sharepoint/docs-conceptual/sharepoint-online/connect-sharepoint-online.md:74]:
```suggestion
> There is a known issue between the SharePoint Online Management Shell module and SharePoint Client Components SDK where the module will fail to load if both are installed on the same computer. If you encounter this issue and you are not trying to use the SharePoint Online Management Shell module in a SharePoint Server, uninstall the SharePoint Client Components SDK.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@get-itips copy review. @tseward please comment

File: sharepoint/docs-conceptual/sharepoint-online/connect-sharepoint-online.md (hunk 1 above)
Comment (tseward) [inline sharepoint/docs-conceptual/sharepoint-online/connect-sharepoint-online.md:74]: > There is a known issue between the SharePoint Online Management Shell module and SharePoint Client Components SDK where the module will fail to load if both are installed on the same computer (excluding a computer running SharePoint Server). If you encounter this issue uninstall the SharePoint Client Components SDK.

**Review by tseward — CHANGES_REQUESTED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by dariomws — COMMENTED** [review]
I'm not sure if we can add this information, to be honest. I cannot approve.

**Comment by yogkumgit** [comment]
@get-itips and @tseward Please discuss it with Dario and resolve.

**Comment by get-itips** [comment]
Closing as no consensus for approval was achieved.

**Pull request closed without merging**